onelogin_aws_cli/configuration.py

```diff
@@ -7,24 +7,31 @@
 class ConfigurationFile(configparser.ConfigParser):
     """Represents a configuration ini file on disk"""
 
-    DEFAULTS = dict(save_password=False)
+    DEFAULTS = dict(
+        save_password=False,
+        duration_seconds=3600
+    )
 
     def __init__(self, config_file=None):
-        super().__init__(default_section='defaults')
-
-        self[self.default_section] = dict(self.DEFAULTS)
+        super().__init__(
+            default_section='defaults',
+        )
 
         self.file = config_file
 
         if self.file is not None:
             self.load()
 
+    @property
```

Comment:
All this jazz is no longer needed since we're never updating defaults with the fallbacks.